packages/IMAPClient-Tests.package/ICEmailTest.class/instance/testGetDateFormatted.st

```diff
@@ -0,0 +1,13 @@
+accessing
+testGetDateFormatted
+	
+	|previousOffset|
+	
+	previousOffset := DateAndTime localOffset.
+	
+	"use Mitteleuropäische Sommerzeit"
```

Comment:
Since `Mitteleuropäische Sommerzeit` refers to something local, i wrote it as a local term. So i could go for 
1. `"use "Mitteleuropäische Sommerzeit"`
2. `use central European Summer Time`

Reply:
What do you prefer?